### PR TITLE
[#7] Develop WBEM Library

### PR DESCRIPTION
**Pull request by CherfaElyes**
* Developed WBEM Library
* Updated License headers
* Updated the index.md
* Updated Sentry icon

### CODE REVIEW

File: javadoc.txt
Comment (NassimBtk) [inline javadoc.txt:1]: Remove.

File: pom.xml [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 	<groupId>org.sentrysoftware</groupId>
-	<artifactId>MYARTIFACT</artifactId>
-	<name>MY PROJECT</name>
-	<version>0.1.00-SNAPSHOT</version>
-	<description>SOME DESCRIPTION</description>
+	<artifactId>wbem</artifactId>
+	<name>WbeM Java Client</name>
+	<version>1.0.00-SNAPSHOT</version>
+	<description>WbeM Java Client</description>
   <packaging>jar</packaging>
```
Comment (NassimBtk) [inline pom.xml:9]: Indent using tabulations.

File: pom.xml [hunk 2]
```diff
@@ -95,6 +114,17 @@
 			<artifactId>junit-jupiter-engine</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```
Comment (NassimBtk) [inline pom.xml:117]: Remove this dependency.

File: pom.xml [hunk 3]
```diff
@@ -112,6 +142,7 @@
 						<source>${maven.compiler.target}</source>
 						<show>public</show>
 						<notimestamp>true</notimestamp>
+						<excludePackageNames>org.sentrysoftware.wbem.javax.*:org.sentrysoftware.wbem.sblim.*</excludePackageNames>
```
Comment (NassimBtk) [inline pom.xml:145]: 🤣

**Review by NassimBtk — CHANGES_REQUESTED** [review]
Minor changes required and we are good.

File: pom.xml [hunk 4]
```diff
@@ -2,18 +2,18 @@
 	<modelVersion>4.0.0</modelVersion>
 
 	<groupId>org.sentrysoftware</groupId>
-	<artifactId>MYARTIFACT</artifactId>
-	<name>MY PROJECT</name>
-	<version>0.1.00-SNAPSHOT</version>
-	<description>SOME DESCRIPTION</description>
-  <packaging>jar</packaging>
+	<artifactId>wbem</artifactId>
+	<name>WbeM Java Client</name>
```
Comment (NassimBtk) [inline pom.xml:6]: `<name>WBEM Java Client</name>`

File: pom.xml [hunk 5]
```diff
@@ -2,18 +2,18 @@
...
+	<version>1.0.00-SNAPSHOT</version>
+	<description>WbeM Java Client</description>
```
Comment (NassimBtk) [inline pom.xml:8]: `<description>WBEM Java Client</description>`

File: src/main/java/org/sentrysoftware/wbem/client/Utils.java [hunk 6]
```diff
@@ -0,0 +1,106 @@
+package org.sentrysoftware.wbem.client;
+
+/*-
+ * ╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲
+ * WbeM Java Client
```
Comment (NassimBtk) [inline src/main/java/org/sentrysoftware/wbem/client/Utils.java:5]: WBEM Java Client

**Review by NassimBtk — CHANGES_REQUESTED** [review]

File: src/site/markdown/index.md [hunk 7]
```diff
@@ -1,3 +1,70 @@
-# Documentation
+# WBEM Java Utility
```
Comment (NassimBtk) [inline src/site/markdown/index.md:1]: WBEM Java Client

**Review by NassimBtk — CHANGES_REQUESTED** [review]

File: src/site/markdown/index.md (hunk 7 above)
Comment (CherfaElyes) [reply src/site/markdown/index.md:1]: I'll check if other documents are impacted by the "WbeM" curse

File: src/it/java/org/sentrysoftware/wbem/WbemIT.java [hunk 8]
```diff
@@ -0,0 +1,148 @@
+package org.sentrysoftware.wbem;
```
[the remark this answers is not in the document]
Comment (bertysentry) [reply src/it/java/org/sentrysoftware/wbem/WbemIT.java]: Or we release our emulator in open source, ha ha ha 😁

**Review by NassimBtk — APPROVED** [review]

**Pull request merged**